R/grid_helpers.R

```diff
@@ -205,6 +212,12 @@ make_rename_arg <- function(grid, model, submodels) {
   res
 }
 
+get_metrics_by <- function(metric_set) {
```

Comment:
You got it! [33a7000](https://github.com/tidymodels/tune/pull/684/commits/33a7000b6edbd5a81a749f909d37a8d8009e5125). This will be helpful for the `autoplot` functionality, as well.